.github/workflows/create-action-registry.yml

```diff
@@ -0,0 +1,73 @@
+name: Create Action registry
+"on":
+    pull_request:
+        paths:
+            - registry/**
+    push:
+        branches:
+            - main
+        paths:
+        - registry/**
+    release:
+        types:
+            - published
+jobs:
+    create-action:
+        name: Create Action
+        runs-on:
+            - ubuntu-latest
+        steps:
+            - if: ${{ github.event_name != 'pull_request' || ! github.event.pull_request.head.repo.fork }}
+              name: Docker login ghcr.io
+              uses: docker/login-action@v1
+              with:
+                password: ${{ secrets.IMPLEMENTATION_GHCR_TOKEN }}
+                registry: ghcr.io
+                username: ${{ secrets.IMPLEMENTATION_GHCR_USERNAME }}
```

Comment:
where do these `IMPLEMENTATION_GHCR_*` secrets come from? Is this a standard GHA thing?

Reply:
They're secrets in that I've [added to our org](https://github.com/organizations/buildpacks/settings/secrets).  The could've been named anything, but GHCR tokens for the implementation team in this case.